recipes/freefem-feedstock/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "FreeFem" %}
+{% set version = "4.10" %}
+{% set sha256 = "957994c8f24cc2a671b8c116ae530796c3a431d4157ee71a3d6aab7122e7570d" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}_{{ version }}.tar.gz
+  url: https://github.com/{{ name }}/{{ name }}-sources/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [py2k or osx or win]
+  missing_dso_whitelist:
+    - ffnewuoa.so
+    - libhdf5.so.103
+
+requirements:
+  build:
```

Comment:
I tried this, but failed. As soon as I move those libraries into the host section it stops compiling : the library is required at compile time, not only at link time. (It can be easily shown by moving a library into the host section)

Reply:
The linker should be searching `$PREFIX/lib`, but it's only searching `$BUILD_PREFIX/lib`. That's why it can't find any of the libraries listed in the host section.

Reply:
I think (hope) that I could resolve all of this (thanks for all the hints!), marking as resolved